querybook/webapp/lib/stream.ts

```diff
@@ -48,6 +48,18 @@ export class DeltaStreamParser {
         return { ...this._result };
     }
 
+    public getResult(ended: boolean): { [key: string]: string } {
```

Comment:
I am a bit iffy about having mutation in getter. What about just rename this to ?
```
public close() { ... }

```
then at the end you do:
```
parser.close();
onStreamingEnd?.(parser.result);
```
also change result into a getter like
```
public get result() { return {...this._result} }
```
then the parse code can be:
```
parser.parse(delta);
onStreaming(parser.result);
```